src/internal/macros.rs

```diff
@@ -157,10 +157,11 @@ macro_rules! bitflags {
             )*
         }
     ) => {
+        $(#[$outer])*
+        $vis struct $BitFlags($T);
+
         bitflags::bitflags! {
-            #[derive(Copy, Clone, Debug, Eq, Hash, PartialEq)]
```

Comment:
Why remove this auto-derive of traits? You missed decorating `InteractionResponseFlags` with `#[derive(TypeSize)]` but also didn't add the other derived traits back to it.

Reply:
I removed the auto derive as it is against the documentation of the macro (it doesn't just forward through) and is more clear when implemented on each type. I'll fix that mess up with InteractionResponseFlags.

Reply:
In what sense does it not forward through?

Reply:
It does, but it also adds the derives undocumented.

Reply:
I also didn't miss decorating InteractionResponseFlags, it's just not used anywhere apparently, looks like a separate PR is in order.

Reply:
> It does, but it also adds the derives undocumented.

You mean in the sense that the definition site of the type inside serenity doesn't include these? I guess that makes sense.